gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -230,11 +230,25 @@ MM_VerboseHandlerOutput::handleInitializedRegion(J9HookInterface** hook, uintptr
 	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
 	MM_VerboseWriterChain* writer = _manager->getWriterChain();
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+#if defined(OMR_GC_VLHGC) && defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	bool isArrayletDoubleMapRequested = _extensions->isArrayletDoubleMapRequested;
+	const char *arrayletDoubleMappingStatus = _extensions->indexableObjectModel.isDoubleMappingEnabled() ? "enabled" : "disabled";
+	const char *arrayletDoubleMappingRequested = isArrayletDoubleMapRequested ? "true" : "false";
+#endif /* OMR_GC_VLHGC && OMR_GC_DOUBLE_MAP_ARRAYLETS */
```

Comment:
I think you can remove 'defined(OMR_GC_VLHGC) &&' 

isVLHGC() is always defined and handles the compile flag internally:

```
MMINLINE bool
	isVLHGC()
{
#if defined(OMR_GC_COMBINATION_SPEC)
		return 0 != _isVLHGC;
#elif defined(OMR_GC_VLHGC)
		return true;
#else
		return false;
#endif
}
```